runtime/lib/ttnn/operations/creation/arange.cpp

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "arange.h"
+#include "tt/runtime/detail/logger.h"
+#include "tt/runtime/detail/ttnn.h"
+#include "tt/runtime/detail/workarounds.h"
+#include "tt/runtime/ttnn/operations/utils.h"
+#include "tt/runtime/ttnn/utils.h"
+#include <ttnn/operations/moreh/moreh_arange/moreh_arange.hpp>
+
+namespace tt::runtime::ttnn::operations::creation {
+void run(const ::tt::target::ttnn::ArangeOp *op, ProgramContext &context) {
+  ProgramTensorPool &tensorPool = context.getTensorPool();
+  //   DeviceVariant targetDevice =
+  //       context.getTargetDevice(op->device()->global_id());
+  ::ttnn::Tensor out = ::ttnn::arange(op->start(), op->end(), op->step(),
```

Comment:
These creation ops are a little trickier. We need to handle multi-chip creation as well. You can take a look at how it's done in `creation/full.cpp`. If you have any questions ping me!

Reply:
@jnie-TT is this a temporary solution? My concern is that emitc is never going to work for multidevice if we do it like this.

@nsmithtt